src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ConsentOrderApprovedController.java

```diff
@@ -93,6 +99,7 @@
                 .consentOrder(consentOrderAnnexStamped);
 
             if (featureToggleService.isApprovedConsentOrderNotificationLetterEnabled()) {
+                log.info("Adding approvedConsentOrderNotificationLetter to approvedOrderBuilder");
```

Comment:
might be too much logging?

Reply:
CAN NEVER HAVE TOO MUCH LOGGING - but yes totally agree, just doing so for QA so we can be 100% confident in our approach, will take out the non-essential logging once we have verification it works as expected